src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -1475,9 +1475,12 @@ If such a class is used as a base class and if the deriving class defines a dest
   <data name="ERR_DuplicateInterfaceInBaseList" xml:space="preserve">
     <value>'{0}' is already listed in interface list</value>
   </data>
-  <data name="ERR_DuplicateInterfaceWithTupleNamesInBaseList" xml:space="preserve">
+  <data name="ERR_DuplicateInterfaceWithTupleNames" xml:space="preserve">
     <value>'{0}' is already listed in the interface list on type '{2}' with different tuple element names, as '{1}'.</value>
```

Comment:
>is already listed in the interface list on type [](start = 17, length = 47)

Is this always accurate? Would it be more accurate to say "is already implemented by type"?